controllers/storageclient_controller.go

```diff
@@ -526,98 +463,61 @@ func (s *StorageClientReconciler) reconcileClientStatusReporterJob(instance *v1a
 	return reconcile.Result{}, nil
 }
 
-func (s *StorageClientReconciler) list(obj client.ObjectList, listOptions ...client.ListOption) error {
-	return s.Client.List(s.ctx, obj, listOptions...)
+func (r *StorageClientReconciler) list(obj client.ObjectList, listOptions ...client.ListOption) error {
+	return r.Client.List(r.ctx, obj, listOptions...)
 }
 
-func (s *StorageClientReconciler) reconcileDefaultStorageClaims(instance *v1alpha1.StorageClient) error {
-
-	if instance.GetAnnotations()[defaultClaimsProcessedAnnotationKey] == "true" {
-		// we already processed default claims for this client
-		return nil
-	}
-
-	// try to list the default client who is the default storage claims owner
-	claimOwners := &v1alpha1.StorageClientList{}
-	if err := s.list(claimOwners, client.MatchingFields{defaultClaimsOwnerIndexName: "true"}); err != nil {
-		return fmt.Errorf("failed to list default storage claims owner: %v", err)
-	}
-
-	if len(claimOwners.Items) == 0 {
-		// no other storageclient claims as an owner and take responsibility of creating the default claims by becoming owner
-		if utils.AddAnnotation(instance, defaultClaimsOwnerAnnotationKey, "true") {
-			if err := s.update(instance); err != nil {
-				return fmt.Errorf("not able to claim ownership of creating default storageclaims: %v", err)
-			}
-		}
-	}
+func (r *StorageClientReconciler) reconcileStorageClaims() error {
```

Comment:
not that this func became trivial, can we please move the 2 calls into the call site and rename them from `create` to `reconcile`?

Reply:
done.